### PR TITLE
Fixed 1.10.2 Localisation (Without Breaking Stuff)

### PR DESCRIPTION
**Pull request by SnowShock35**
- CHANGED: Updated pack_format to version 2 in pack.mcmeta file
- CHANGED: Updated version string in build.gradle and main class
- CHANGED: Re-capitalized the language files to fix foremly broken
localisation

Amended PR - Original #204

### CODE REVIEW

**Comment by SnowShock35** [comment]
@Xalcon There we go. Properly done. Apologies for making so many mistakes. Not my area of expertise 😛

File: build.gradle [hunk 1]
```diff
@@ -13,7 +13,7 @@ buildscript {
 apply plugin: 'net.minecraftforge.gradle.forge'
 
 
-version = "1.10.2-1.3.11" 
+version = "1.10.2-1.3.12"
```
Comment (Xalcon) [inline build.gradle]: I'm not sure what azanors stand on this is, but imo external pull requests shouldnt change the project version number. Thats the job of the maintainer and should only be done if there is a new build.

File: src/main/resources/changelog.txt [hunk 2]
```diff
@@ -1,3 +1,6 @@
+1.3.12
+- fixed localization issues (again)
+
```
Comment (Xalcon) [inline src/main/resources/changelog.txt]: Same as the version thing - should be done by the maintainer (imo!)

**Comment by Xalcon** [comment]
@SnowShock35 Im just a random guy helping out; if anyone deserves an apology, its azanor.

File: src/main/java/baubles/common/Baubles.java [hunk 3]
```diff
@@ -33,7 +33,7 @@
 	
 	public static final String MODID = "Baubles";
 	public static final String MODNAME = "Baubles";
-	public static final String VERSION = "1.3.11";
+	public static final String VERSION = "1.3.12";
```
Comment (Xalcon) [inline src/main/java/baubles/common/Baubles.java]: See comment from `build.gradle`

**Comment by SnowShock35** [comment]
@Xalcon I had done the version number changes and change log changes simply to save time for azanor.

Also, I've already apologized to azanor. I was simply apologizing to you for not knowing how it should or should not be done. Also, I don't mean to be funny, but at least I made the effort to at least try and fix what was broken. 😕

**Comment by Xalcon** [comment]
I've cloned the latest 1.10.2 from this (azanors) repository and merged the pull request locally and your changes seem to work. Not sure why translations were broken before the 1.11.2 changes were made to the 1.10.2. The lang files had the [correct format before](https://github.com/Azanor/Baubles/commit/0308bf03eb38e498175db544443396f27804ab3d) but it didnt work according to #181 

Anyway, looks good now :)

**Pull request merged**